fava/static/sass/_base.scss

```diff
@@ -595,3 +630,36 @@ table.queryresults {
         margin-left: 3px;
     }
 }
+
+/* Modifications for ipad */
```

Comment:
I used the chrome inspector to emulate the various phone resolutions and the only one that was this large was ipad.  I can add more comments explaining myself though.